Formula/poppler.rb

```diff
@@ -76,7 +76,7 @@ def install
       [
         "#{lib}/libpoppler-cpp.dylib",
         "#{lib}/libpoppler-glib.dylib",
-        "#{lib}/libpoppler-qt5.dylib",
+        "#{lib}/libpoppler-qt6.dylib",
```

Comment:
This might make future version bumps easier
```suggestion
        "#{lib}/libpoppler-qt#{Formula["qt"].version.major}.dylib",
```

Reply:
it will be a while until qt7 comes up ...

Reply:
True, but doing it that way means you (or anyone else) can ensure consistency when changing the "qt" dependency, just by doing a global search-and-replace (e.g. `sed -i 's/"qt"/"qt@5.7"/g Formula/poppler.rb`).

Reply:
however there is still the cmake option. 
In this case, I am a proponent of KISS (keep it simple...)

Reply:
Yes. The suggestion here makes the formula much simpler to maintain.